tests/virtualization/yast_virtualization.pm

```diff
@@ -8,15 +8,22 @@ sub run() {
     wait_idle;
     send_key "alt-f10";
     become_root;
-    script_run "yast2 virtualization";
-    wait_idle;
-    assert_screen "virt-sle12sp1-gnome_yast_virtualization", 50;
-    # select everything
-    send_key "alt-x", 10;    # XEN Server
-    send_key "alt-e", 10;    # Xen tools
-    send_key "alt-k", 10;    # KVM Server
-    send_key "alt-v", 10;    # KVM tools
-    send_key "alt-l", 10;    # libvirt-lxc
+    script_run("/sbin/yast2 virtualization; echo yast2-virtualization-done-\$? > /dev/$serialdev", 0);
+    if (check_var("FLAVOR", "Desktop-DVD")) {
+        assert_screen "virt-sled12sp2-gnome_yast_virtualization";
```

Comment:
This seems to be 2 different tests in one - one for server and one for desktop, they do completely different things ;(

But even worse - you hardcode service packs in tag names, which I find very confusing. But I see this was @aginies 's choice - but possibly you should correct it instead of building on top of it. Because what I see in the needles is not Sp1 or Sp2 specific - but I don't know much about it.